docs/sources/concepts/binding.md

```diff
@@ -69,11 +74,20 @@ this.BindCommand(ViewModel, x => x.OkCommand, x => x.RootView, "KeyUp");
 
 Direct bindings between properties are convenient, but often the two types are
 not assignable to each other. For example, binding an "Age" property to a
-TextBox would normally fail, because TextBox expects a string value. Instead,
+TextBox would normally fail, because the TextBox expects a string value. Instead,
 ReactiveUI has an extensible system for coercing between types.
 
-See the details about Binding Type Converters in the "Customization" section
-for more information about how to extend property type conversion.
+For simple one-way bindings, you can use the conversion parameter in the 
+`OneWayBind` method.
+
+```cs
+
+// Note: Age is an integer
+this.OneWayBind(ViewModel, x => x.Age, x => x.Name.Text, x => x.ToString()); // In the last parameter, we .ToString() the integer
```

Comment:
Perhaps mention here that this last parameter is a `Func<int,string>` converter, rather than talk about all the implementation details?